<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QDialog" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome!</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="welcomelabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>(Please click &quot;cancel&quot; button in the muscle info on the left before clicking another muscle.)</string>
   </property>
  </widget>
  <widget class="QLabel" name="volumeTxt">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>40</y>
     <width>186</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Volume-          Volume +</string>
   </property>
  </widget>
  <widget class="QSlider" name="sliderVolume">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>60</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>710</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Use this slider to turn the model to see the side and back!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="muteButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>40</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hardcoreStyleButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Hardcore</string>
   </property>
  </widget>
  <widget class="QPushButton" name="relaxStyleButton">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>10</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Relax</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
